scripts/peer-dependency-generator/findAndReplace.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-console */
```

Comment:
i assume we will move these to a common location in the future?

Reply:
Yeah, once we decided where we want these common scripts we use in the monorepos to live, we'll DRY them up in a single spot.